examples/dedupe/worker.py

```diff
@@ -1,11 +1,9 @@
 import asyncio
-import random
 
 from dotenv import load_dotenv
 
 from hatchet_sdk import Context, Hatchet
 from hatchet_sdk.clients.admin import DedupeViolationErr
-from hatchet_sdk.loader import ClientConfig
 
 load_dotenv()
```

Comment:
note for future: i personally dont like this here... i like how npm runs this in the process before running the example

Reply:
yeah, you mean the `load_dotenv`? I think we'd benefit from using Pydantic Settings (Pydantic everywhere!) in the examples: https://docs.pydantic.dev/latest/concepts/pydantic_settings/#dotenv-env-support